__tests__/__utils__/kv.ts

```diff
@@ -30,6 +30,11 @@ export function createKV<Key extends string>(): KV<Key> {
     // eslint-disable-next-line @typescript-eslint/require-await
     async put(key: Key, value: string, _?: { expirationTtl: number }) {
       values[key] = value
+      expect(key.length).toBeLessThanOrEqual(512)
```

Comment:
This does not work since `expect()` can only be written inside a `test()` function. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Control_flow_and_error_handling#Exception_handling_statements for how throwing an exception.